core/src/util/usLDAPFilter.cpp

```diff
@@ -0,0 +1,131 @@
+/*=============================================================================
```

Comment:
I searched online, and found that it seems not possible with github UI. 
Summary of changes:
- use usProperties class instead of usServicePropertiesImpl class.
- Added "Match(const Bundle& bundle)" method to enable LDAP query on bundle properties